engine/src/main/resources/templates/Patch/dailyEmailBody.html

```diff
@@ -0,0 +1,243 @@
+                  </h2>
+                </div>
+                <div>
+                  <span class="rh-metric__count">{action.context.patch.security.unique_system_count}</span>
+                </div>
+              </div>
+            </div>
+            <!-- end 50 col -->
+            <!--[if (gte mso 9)|(IE)]>
+                    </td>
+                    </tr>
+                    </table>
+                    <![endif]-->
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <br>
+            <div class="container" style="display: table;">
+              <div class="rh-card" style="margin-right: 5px; width: 48%;">
+                <div class="rh-card__header">
+                  <h2>
+                    Systems affected by Bugfix Advisories
+                  </h2>
+                </div>
+                <div>
+                  <span class="rh-metric__count">{action.context.patch.bugfix.unique_system_count}</span>
+                </div>
+              </div>
+              <div style="display: table-cell; padding-right: 12px"></div>
+              <div class="rh-card" style="margin-left: 12px; width: 48%; height: 100%;">
+                <div class="rh-card__header">
+                  <h2>
+                    Systems affected in Total
+                  </h2>
+                </div>
+                <div>
+                  <p></p>
+                  <span class="rh-metric__count">{action.context.unique_system_count}</span>
+                </div>
+              </div>
+            </div>
+            <!-- end 50 col -->
+            <!--[if (gte mso 9)|(IE)]>
+                    </td>
+                    </tr>
+                    </table>
+                    <![endif]-->
+          </td>
+        </tr>
+        <!-- end content block -->
+      </tbody>
+    </table>
+    <br>
+  </td>
+</tr>
+<!-- end body section -->
+
+<!-- Body section -->
+<tr>
+  <td class="rh-body">
+    <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0"
+      width="100%">
+      <tbody>
+        <!-- Content block -->
+        <tr>
+          <td class="rh-content__block">
+            <p>
+              <!-- Hi {{user.first_name}},
+                        <br>
+                        <br> -->
+              Hi,
+              <br>
+              <br>
+              Here is your Patch advisories summary affecting your systems for {action.context.start_time.toStringFormat()}.
+            </p>
+          </td>
+        </tr>
+        <!-- end content block -->
+
+        <!-- Content block -->
+        <tr>
+          <td class="rh-content__block">
+            <table role="presentation" border="0" cellpadding="0" cellspacing="0" align="center"
+              class="rh-button-table">
+              <tr>
+                <td class="rh-cta-link" align="center">
+                  <a target="_blank" class="rh-url" href="https://console.redhat.com/insights/patch/advisories">
```

Comment:
You can use the environment helper to generate the url depending the environment (stage/prod)

```suggestion
                  <a target="_blank" class="rh-url" href="{environment.url()}insights/patch/advisories">
```